### DISCUSSION TITLE
An Apology to the community

### DISCUSSION THREAD

**Discussion by Zitchas** [endless-sky/endless-sky#10031] [Announcements]
Rehashing a discussion about my relationship with the Devs in the vision discussion reminded me that I offered, as part of those discussions, to apologize for mistakes made in the past. I originally made that offer as part of the negotiations about an agreement to putting the past behind us and working together cooperatively once again; but having remembered that, it occurs to me that there is no need to wait or put restrictions on that. So here it is, unprompted, unasked for, and with no expectations.

---

I would just like to acknowledge that over the past two years since MZ decided to make me an admin I have made a variety of mistakes and poor choices. Those mistakes include how I handled the aftermath of that event, and poorly handled conversations since then. The results of these have caused some problems, and a lot of tension between various people, and directly harmed game development. I would love to be able to go back in time to change those decisions and alter the path of those conversations, but I cannot. So at this point, I would like to offer my heartfelt apology to the community in general for those mistakes, and to those who were harmed by my actions and words in particular. I do not know who all of you are, but I know it happened nonetheless. I am sorry for the feelings I have hurt, for the offenses I have given, and the mistakes made. I hope that someday, you may find it in your heart to forgive me.

I cannot unmake my mistakes, but I can and will try to do better. I cannot promise to be perfect, or do everything that everyone asks of me; but I can commit to making the effort.

---

I would also like to express my hope that we can patch the rifts and continue to work together. If I didn't have that hope, I would have long since abandoned this project. Every action I take here, every post, is a testament to that hope. Hoping that someday tension can be replaced by enjoyment; that conflict can be replaced by cooperation. I will continuing hoping that someday these hopes becomes fact.

I do not know the path forward, but I remain hopeful that one exists.

Thank you for reading.

Zitchas

**Comment by samrocketman** [comment]
I think it needs to be said: at the end of the day MZ intent that the project remain open and available to more contributions has been preserved.

I'm not 100% certain about the rift. But in terms of shepherding the project to remain open; it still is.

I think it takes a lot of integrity to be an admin and even with your elevated access it was never abused.

**Reply by Zitchas** [reply]
Thank you.

**Comment by xX-Dillinger-Xx** [comment]
I'm still trying to sift through the large amount of recent discussion in regards to the "Vision Document". I commend you for trying to mend the broken fences but felt I needed to point something out.
 
I don't feel that this is all on you. We all know, there are two distinct camps surrounding the development of ES. Both sides have made inappropriate and reactionary comments fueled solely by allegiances to one side or the other. ~~The problem is, currently only one side has the power to influence the development. I've only been here for a few months but I can already predict which way a discussion is going to go based on who starts it and who responds first.~~ EDIT: I was going to delete this but don't want to hide my mistakes. I have been accused of making unfounded comments before and have done it again. I don't really know the actual number of supporters for each "camp" and any perceived imbalance in influential powers is nothing more than an assumption.
 
This community is a hostile place. There are many factors that are causing this. Ego is a big one. Too big of an ego creates closed ears. Another big one is frustration. It is natural to be overly defensive of something that you are passionate about, especially if you just spent untold hours of time developing the idea. One more issue is lazy thinking. What I mean by this is, lazy thinkers are people who just blindly agree with something or someone without taking the time to think about what is being presented. We have all seen this. Ideas get prematurely shut down or accepted before the discussion has even concluded. I'm not just referring to the Dev's. Prominent members of the community do this as well. They adamantly support one side or the other without taking time to fully understand the topic.

**Reply by brendanjones** [reply]
I’m just going to point out that, as someone not in the core team and only intermittently contributing, I have seen exactly zero hostility. Quite the opposite, the discussions here are open and constructive, and everyone is helpful. Maybe I’m just oblivious but this repo has always felt like a welcoming place to me, which is more than I can say about most open source repos. 

Whatever rifts or bad feelings there are, maybe those people just need to step back and see the big picture: this is a game and we’re all here to have fun so don’t take it too seriously.

**Reply by xX-Dillinger-Xx** [reply]
I'm not sure what was so confusing in my comment, but it doesn't really matter at this point. Any further explanation would be a waste of time.
@brendanjones 
You are correct that you have been accepted and welcomed. That's great to see. However, I have over stayed my welcome. It has been pointed out to me, that it was all on me. Sometimes, some people just can't fit in and clearly that is the case for me.

**Comment by kenbadertscher** [comment]
I will say that I'm new to Endless Sky.  I downloaded the game only last month.  I saw that it was an open source project, and I loved Escape Velocity when I was a kid, so I wanted to hop in and see what I could do to contribute.

What I found in Endless Sky is a truly enjoyable game, and I have found nothing about the game that I've observed and said "I need to help fix that right away."  

I say this only to point out that whatever rift there is in the developer community that I have not read about or been exposed to, know that it has not manifested itself in the game from a newcomer's perspective.  For any differences that may exist, the outcome has been a game that I've enjoyed playing quite a lot.  

So Zitchas, I encourage you to see the forest, and not stay so focused on the trees.  Thank you to you, and to all of the contributions of the community over the years that has created the Endless Sky that exists today.  I've truly enjoyed playing it.  It personally meant a lot to me to be able to play the next best thing to Escape Velocity, because I give EV credit for setting me down the software development career path I have today.  Your efforts have meant more than you ever could know to this particular stranger.

**Comment by rovermicrover** [comment]
> This community is a hostile place. There are many factors that are causing this. Ego is a big one. Too big of an ego creates closed ears. Another big one is frustration. It is natural to be overly defensive of something that you are passionate about, especially if you just spent untold hours of time developing the idea.

It may be at the point where the repo could be forked into specific parts.

The underlying engine as one repo.

Then the story and content as another. People would then be free to fork the content anyway they want or please and PR anything back to the primary as it where as they want. This shouldn't actually be that hard given all the recent work on cleaning up the build process and how well the github actions run. Hell I have contributed to the C++ code without  ever having compiled on my local, instead just using the github action artifcats.

This is an open source project, either people fork when vision diverge or shit just keeps getting toxic until the project dies.
